app/cisco/ccna/automation-cisco-platforms-and-development/page.css

```
@@ -0,0 +1,484 @@
+  --bg-code: #0a1424;
+  --border: rgba(124, 158, 255, 0.18);
+  --border-soft: rgba(124, 158, 255, 0.1);
+  --accent: #7c9eff;
+  --accent-2: #ffd479;
+  --accent-2-soft: rgba(255, 212, 121, 0.14);
+  --text: #e8edf7;
+  --text-muted: #9aa8c4;
+  --text-faint: #6b7a99;
+  --radius: 14px;
+  --sidebar-width: 300px;
+  --font-sans: 'Noto Sans JP', 'Inter', -apple-system, BlinkMacSystemFont, 'Segoe UI', sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)